distribution/pom.xml

```diff
@@ -36,6 +36,20 @@
         <dependency>
             <groupId>org.wso2.ei</groupId>
             <artifactId>org.wso2.micro.integrator.server</artifactId>
+            <exclusions>
```

Comment:
Let's remove these from org.wso2.micro.integrator.server.feature itself

Reply:
Done in 838be769aaa067edf0f4aa07c78faa6d3dd200ba